src/app/core-components/welcome/welcome.component.ts

```diff
@@ -33,4 +84,11 @@ export class WelcomeComponent {
   openWizardModal() {
     this.sharedDataService.createAssessmentModalOpen.next(true);
   }
+
+  goToVisit(visit: IdbOnSiteVisit) {
+    this.companyIdbService.setSelectedFromGUID(visit.companyId);
+    this.facilityIdbService.setSelectedFromGUID(visit.facilityId);
+    this.onSiteVisitIdbService.setSelectedFromAssessmentGUID(visit.assessmentIds[0]);
```

Comment:
Is `visit.assessmentIds[0]` a concern if no assessments have been created for the visit?
Currently, it seems that user is allowed to set up a new visit without creating any assessment

Reply:
updated thanks